RickLeinecker/Fortuna
# Set 3v3 wager now works on front and back end

**Description**
The armory has been updated to handle setting two wagers: a 1v1 wager and 3v3 wager. The Setup button now allows users to pick their tank in the SetWagerPopup. 

In the popup, they can switch between whether they want to set a 1v1 wager or 3v3 wager by clicking on buttons in the popup. The 1v1 wager cannot have a null tank, while the 3v3 wager can have up to two.

On the backend, when we unfavorite a tank team, we set the user's `favoriteTanks = [null, null, null]`.

I still need to implement the 3v3 in the battle arena (which shouldn't take long hopefully) and make sure the new wager setting system works in there. That will be a separate PR.

**Resolves These Issues**
Resolves none, yet

**Type of change**
Check options that are relevant.


**Steps to Verify Functionality**
1. Create a new account.
2. Go to armory.
3. View new wager section in the bottom left.
4. Click setup.
5. Click 1 v 1 button.
6. Choose a tank and a wager > 50.
7. Click wager button.
8. Check that the favoriteTank field and wager field has been set.
9. Click setup again.
10. Click 3 v 3 button.
11. Set up a full team of tanks (no No Tank option selected).
12. Choose a wager > 50.
13. Click wager.
14. Check that the favoriteTanks field and wager3v3 field has updated.
15. Click the remove button.
16. Click 1 v 1 button.
17. Click remove.
18. Check that favoriteTank is null and wager is 0.
19. Repeat to remove the 3 v 3 wager.

**Final Checklist:**
Go down the list and check them off.
